### DISCUSSION TITLE
windows support

### DISCUSSION THREAD

**Discussion by sudipta1g** [givanz/Vvveb#43] [General]
Hello,

I am newbie for making webpages.
Is possible to install on a windows machine?

Is it possible make few web pages from your demo webiste?

**Comment by givanz** [comment]
Hi

You can test any function on the demo website but saving is disabled.

You can use xampp to install on windows https://www.apachefriends.org/download.html

But it's easier to create a free hosting account https://www.infinityfree.com install it there and create your own site.

After creating the hosting account use ftp to upload the files from the zip to htdocs folder then follow the installation instructions.
